kai_mcp_solution_server/tests/mcp_loader_script.py

```
@@ -0,0 +1,293 @@
+    server_path: str | None = None
+    mount_path: str | None = None
+    full_output: bool = False
+    verbose: bool = False
+    insecure: bool = False
+
+
+original_ssl_create_default_context = ssl.create_default_context
+
+
+@asynccontextmanager
+async def create_client(args: MCPClientArgs) -> AsyncIterator[ClientSession]:
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)